alby-nostr-wallet-connect/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.7"
+services:
+  app_proxy:
+    environment:
+      APP_HOST: alby-nostr-wallet-connect_server_1
+      APP_PORT: 8080
+      PROXY_AUTH_ADD: "false"
+  server:
+    image: ghcr.io/getalby/nostr-wallet-connect:0.1.0@sha256:7e3b59af4abdcbee5def7648ba5253c68e638647bc4a407974a61d13aa2b3b3d
+    volumes:
+      - ${APP_DATA_DIR}/data:/data
```

Comment:
We need to add the '${APP_DATA_DIR}/data' directory to this repository so that it exists on the user's host machine before Docker Compose is run. When Docker Compose is executed, if the '${APP_DATA_DIR}/data' directory does not exist, Docker will automatically create it as root, while we want to have it owned by the umbrel user (1000). 

Check out [Tailscale](https://github.com/getumbrel/umbrel-apps/tree/master/tailscale) as a direct example of what you can implement. Just create a folder called `data` inside `alby-nostr-wallet-connect` and add an empty `.gitkeep` file to ensure that the directory is tracked by Git

Do you know if the NWC app itself needs to run as root? I see that the database is owned by root:

```
$ ls -l ~/umbrel/app-data/alby-nostr-wallet-connect/data
total 72
-rw-r--r-- 1 root root 69632 Jun 23 05:24 nostr-wallet-connect.db
```

Ideally we would have this running as user 1000:1000.

Reply:
I've added the `data` directory, and I don't see why there would be any problems if the app is ran as a non-root user.